content/en/tracing/setup/dotnet-core.md

```diff
@@ -23,7 +23,7 @@ further_reading:
 
 ## Compatibility
 
-The .NET Tracer supports automatic instrumentation on .NET Core 2.1 and 3.1, as well as Microsoft-deprecated versions 2.0, 2.2 and 3.0.  For a full list of supported libraries, visit the [Compatibility Requirements][1] page.
+The .NET Tracer supports automatic instrumentation on .NET 5 (starting with .NET Tracer `1.20.0`), LTS .NET Core versions 2.1 and 3.1, and Microsoft-deprecated .NET Core versions 2.0, 2.2 and 3.0.  For a full list of supported libraries, visit the [Compatibility Requirements][1] page.
```

Comment:
We should keep this in sync with the one in `content/en/tracing/compatibility_requirements/dotnet-core.md` (which doesn't mention "Microsoft-deprecated versions).